tests/ui/invalid_pymethod_names.stderr

```diff
@@ -0,0 +1,29 @@
+error: name can not be specified for getter
+  --> $DIR/invalid_pymethod_names.rs:10:7
+   |
+10 |     #[name = "num"]
+   |       ^^^^
+
+error: name can not be specified multiple times
+  --> $DIR/invalid_pymethod_names.rs:18:7
+   |
+18 |     #[name = "bar"]
+   |       ^^^^
+
+error: name can not be specified with #[new]
+  --> $DIR/invalid_pymethod_names.rs:24:7
+   |
+24 |     #[name = "makenew"]
+   |       ^^^^
+
+error: cannot find attribute `name` in this scope
```

Comment:
`#[name]` attributes should not make it out of the proc-macro even in error conditions. rustc changing error messages for unknown attributes is what's causing the compile failure.

Reply:
:+1: I was thinking I need to refactor this as part of supporting `#[pyfunction]`. The compile failure gives me a solid reason to do it!